wagn/lib/wagn/application.rb

```diff
@@ -56,9 +56,13 @@ def add_gem_path paths, path, options={}
     def config
       @config ||= begin
         config = super
+
         Cardio.set_config config
+        config.autoload_paths += Dir["#{Cardio.gem_root}/mod/*/lib/**/"]
```

Comment:
yeah, it was in the engine and should probably be there.  Still several things to iron out here.  I especially want to figure out how to make it so cardio works without being fed a Railties-specific paths object.